### ISSUE TITLE
Alias for Ti, as time

### ISSUE THREAD

**Issue by lazarusA** [rafaqz/DimensionalData.jl#499]
Currently, when we  ask for dimensions, and time is one of them we get :Ti as the name, is there an automatic way to get instead :time? In general we should get the actual name, and not Ti, which is not at all intuitive and useful.

**Comment by rafaqz** [comment]
Yeah `Ti` is pretty annying. Its because Dates exports `Time` and `T` means type.

We could just ditch using X/Y/Ti type wrappers and use `Dim{:time}` and `time=` keywords. I need to do a review of where the wrappers are actually useful and what to do instead.

Hugely breaking tho

**Comment by rafaqz** [comment]
I kinda like types for `rand(X(5), Ti(10:10:100))` kind of things...

**Comment by lazarusA** [comment]
I just want to get the time symbol if Ti is available 😂. I could just replace it on my workflow and move on with the next step. But it's really odd to have Ti in and doing d[time=1] working even though I don't get the name time when getting dims.

**Comment by rafaqz** [comment]
You can always hack `key2dim` and `dim2key` to make things whatever you want

**Comment by lazarusA** [comment]
> We could just ditch using X/Y/Ti type wrappers and use Dim{:time} and time= keywords. I need to do a review of where the wrappers are actually useful and what to do instead.

coming back to this, maybe yes, it could be better to have `Dim{:time}`, even if its breaking 😆 .
